build/common.sh

```diff
@@ -339,6 +340,27 @@ function kube::build::destroy_container() {
   "${DOCKER[@]}" rm -f -v "$1" >/dev/null 2>&1 || true
 }
 
+# Validate a release version
+#
+# Globals:
+#   VERSION_REGEX
+# Arguments:
+#   version
+# Returns:
+#   If version is a valid release version
+# Sets:
+#  BASH_REMATCH, so you can do something like:
+#    local -r version_major="${BASH_REMATCH[1]}"
+#    local -r version_minor="${BASH_REMATCH[2]}"
+#    local -r version_patch="${BASH_REMATCH[3]}"
+function kube::release::parse_and_validate_release_version() {
+  local -r version="${1-}"
+  [[ "${version}" =~ ${VERSION_REGEX} ]] || {
```

Comment:
I think if this is the only place using the regex, I'd just go ahead and move it into the function.

Also, eventually we're going to have to accept alpha versions. We can fight that battle later, though. (See #13339)